hw/bsp/stm32f3discovery/src/hal_bsp.c

```diff
@@ -155,6 +168,59 @@ hal_bsp_get_nvic_priority(int irq_num, uint32_t pri)
     return pri;
 }
 
+/**
+ * LSM303DLHC sensor default configuration
+ *
+ * @return 0 on success, non-zero on failure
+ */
+int
+config_lsm303dlhc_sensor(void)
```

Comment:
This new function has no prototype and it doesn't seem to be used.

Reply:
This function is used, and it is "declared" in the `pkg.init` section (included in this PR). I didn't remember how this works, but taking a look at the generated code (under `bin/targets/f3_sensors/generated/src/f3_sensors-sysinit-app.c` on my tree):

```
...
    config_pkg_init_stage2();

    /*** Stage 400 */
    /* 400.0: config_lsm303dlhc_sensor (hw/bsp/stm32f3discovery) */
    config_lsm303dlhc_sensor();

    /*** Stage 500 */
    /* 500.0: flash_test_init (test/flash_test) */
    flash_test_init();
...
```

The prototype is also declared in the same file. I did a small change to use `void` as return to match the prototype of all those sysinit funtions.